src/App.js

```diff
@@ -62,7 +76,9 @@ const App = () => {
         <button>Reset Game</button>
       </header>
       <main>
-        <Board squares={squares} />
+        <Board 
+        squares={squares} 
+        onClickCallback= {somefunction}/>
```

Comment:
love that you put your props on separate lines. to make this 100%perfect, we can do this formatting:

```suggestion
        <Board 
            squares={squares} 
            onClickCallback= {somefunction}
        />
```